unlock-app/src/components/creator/CreatorLock.js

```diff
@@ -35,10 +35,9 @@ export class CreatorLock extends React.Component {
   }
 
   render() {
-    // TODO add USD values to lock
     // TODO add all-time balance to lock
 
-    const { lock, transaction, config } = this.props
+    const { lock, transaction, config, conversion } = this.props
```

Comment:
Can you get rid of `conversion` here since it is not used at all?

Reply:
done